backend/src/authentication/authentication.service.ts

```diff
@@ -26,11 +30,30 @@ export class AuthenticationService {
         return `${registrationData.email} ${content.Created}`;
     }
 
+    /**
+      Currently this returns the pin as we need pass that back to frontend.
+      Will be corrected when relevant workflow is introduced.
+     */
+    async registerJunior(registrationData: RegisterJuniorDto): Promise<string> {
+        const userExists = await this.juniorService.getUser(registrationData.phoneNumber);
+        if (userExists) { throw new ConflictException(content.AdminAlreadyExists); }
```

Comment:
`content.AdminAlreadyExists` should probably be renamed as this is quite misleading.

Reply:
Provided the wrong value from the JSON file, updating to provide "content.JuniorAlreadyExists"